test/test_interactive.py

```diff
@@ -135,7 +135,8 @@ def test_sdl2_mixer_music(self, formats, flags, music_name):
 
   def test_sdl2_audio_beeps(self):
     # use closure to check for a possible bug with closure minifying away newer Audio() attributes
-    self.btest_exit(test_file('sdl2_audio_beep.cpp'), args=['-O2', '--closure=1', '--minify=0', '-sDISABLE_EXCEPTION_CATCHING=0', '-sUSE_SDL=2'])
+    # TODO: investigate why this does not pass
```

Comment:
Hmm, that test uses `SDL_OpenAudio`, which is roughly equivalent to `SDL_OpenAudioDevice` with `SDL_AUDIO_ALLOW_ANY_CHANGE`. It also expects U8 and S16 formats to go through unchanged, but the backend has only ever supported F32.

I'm not sure how it was previously passing...

Reply:
Since we don't run these tests with any regularity or record the results its hard to show that it ever passed.